queries/java/highlights.scm

```diff
@@ -94,7 +94,8 @@
   name: (identifier) @type)
 (constructor_declaration
   name: (identifier) @type)
-(type_identifier) @type
```

Comment:
huh, wait you'd want this by default for all type_identifiers, then override it if it's var

Reply:
Hmm yeah I thought so, therefor I pushed a new commit. Please check if it's correct

Reply:
No, I'll just fix it..